README.md

```diff
@@ -74,137 +95,63 @@ cd deephaven-deployment
+| Python   | Yes          | SciKit-Learn        | https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/SciKit-Learn/docker-compose.yml |
+| Python   | Yes          | PyTorch             | https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/PyTorch/docker-compose.yml |
+| Python   | Yes          | TensorFlow          | https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/TensorFlow/docker-compose.yml |
+| Python   | Yes          | NLTK                | https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/NLTK/docker-compose.yml |
+| Groovy   | No           | None                | https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/groovy/docker-compose.yml |
+| Groovy   | Yes          | None                | https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/groovy-examples/docker-compose.yml |
 
-Run the following commands to launch Deephaven for Python server applications, with example data and [NLTK](https://nltk.org/).
 
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/NLTK/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+### Monitor logs
 
-### Launch: Python with example data and PyTorch
+The `-d` option to `docker-compose` causes the containers to run in the background, in detached mode.  This option allows you to use your shell after Docker launches the containers.
 
-Run the following commands to launch Deephaven for Python server applications, with example data and [PyTorch](https://pytorch.org/).
+Since the container is running detached, you will not see any logs. However, you can follow the logs by running:
 
 ```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/PyTorch/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
+docker-compose logs -f
 ```
 
-### Launch: Python with example data and SciKit-Learn
-
-Run the following commands to launch Deephaven for Python server applications, with example data and [SciKit-Learn](https://scikit-learn.org/stable/).
-
-```bash
-curl https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/SciKit-Learn/docker-compose.yml -O
-docker-compose pull
-docker-compose up -d
-```
+Use CTRL+C to stop monitoring the logs and return to a prompt.
 
-### Launch: Python with example data and TensorFlow
+### Authentication
 
-Run the following commands to launch Deephaven for Python server applications, with example data and [TensorFlow](https://www.tensorflow.org/).
+Deephaven, by default, comes stock with pre-shared key to authenticate any users trying to access it. If no key is set, a randomly generated key is printed to the Docker logs. The key can be found by monitoring the logs. It will look like this:
```

Comment:
```suggestion
Deephaven, by default, comes stock with a pre-shared key to authenticate any users trying to access it. If no key is set, a randomly generated key is printed to the Docker logs. The key can be found by monitoring the logs. It will look like this:
```